### ISSUE TITLE
Subcategories / groups for components

### ISSUE THREAD

**Issue by jloh** [ks888/LambStatus#121]
It'd be awesome to be able to group components somehow or have a master component to have other ones under. An example would be for servers, eg:

```yaml
servers:
  - server 1
  - server 2
  - server 3
```

Hopefully this makes sense!

**Comment by ks888** [comment]
Yes, it's awesome to have a feature to group components, though I'm not sure when I can have a time to implement it.

**Comment by rclakmal** [comment]
We would also like to have this feature.

**Comment by Wild1145** [comment]
Just been trying this software out, this would be an invaluable feature 👍
